sample-data/instances/girl-on-the-train.json

```diff
@@ -25,5 +25,8 @@
       "name": "Creator B"
     }
   ],
-  "instanceTypeId": "6312d172-f0cf-40f6-b27d-9fa8feaf332f"
+  "instanceTypeId": "6312d172-f0cf-40f6-b27d-9fa8feaf332f",
+  "natureOfContentTermIds": [
+    "44cd89f3-2e76-469f-a955-cc57cb9e0395"
```

Comment:
Does a nature of content term already exist for this UUID in the standard reference data?

Reply:
yep, it is:
```
{
  "id": "44cd89f3-2e76-469f-a955-cc57cb9e0395",
  "name": "textbook",
  "source": "folio"
}
```

and 
```
{
  "id": "0abeee3d-8ad2-4b04-92ff-221b4fce1075",
  "name": "journal",
  "source": "folio"
}
```
for `aba` and `american-journal-of-medicine`

Reply:
Ok, thanks for checking